packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.types.ts

```diff
@@ -402,3 +401,23 @@ export interface IComboBoxOptionStyles extends IButtonStyles {
    */
   optionTextWrapper: IStyle;
 }
+
+export interface IComboBoxClassNames {

```

Comment:
These should probably have docCategory comments so they show up on the website (especially since they're referenced from existing public interfaces).
```
/**
 * {@docCategory ComboBox}
 */
```

Reply:
I'm not sure really we want to have any of these additional types exposed.

Reply:
resolved